test/unit/spec/common/simpleLogViewer-spec.js

```diff
@@ -0,0 +1,1096 @@
+define([
+    'jquery',
+    'bluebird',
+    'common/simpleLogViewer',
+    'common/jobs',
+    'common/jobCommMessages',
+    'common/jobManager',
+    'common/props',
+    'common/runtime',
+    '/test/data/jobsData',
+    'testUtil',
+], (
+    $,
+    Promise,
+    SimpleLogViewerModule,
+    Jobs,
+    jcm,
+    JobManagerModule,
+    Props,
+    Runtime,
+    JobsData,
+    TestUtil
+) => {
+    'use strict';
+
+    const TEST_POLL_INTERVAL = 10;
+    const { cssBaseClass, SimpleLogViewer, BishopsGame } = SimpleLogViewerModule;
+
+    const { JobManager } = JobManagerModule;
+
+    const jobsByStatus = JobsData.jobsByStatus;
```

Comment:
a fair number of these tests have been taken directly from the old log viewer tests, but I've also added in some new tests.